api/sirsidynix_authentication_provider.py

```diff
@@ -57,7 +57,7 @@ class Keys:
             "label": _("Server URL"),
             "description": _("The external server url."),
             "required": True,
-            "default": "https://vendor1-sym.sirsidynix.net/ilsws_current/",
+            "default": "",
```

Comment:
Can probably just completely drop `default`, instead of setting it to a blank string.